README.md

```diff
@@ -70,15 +70,24 @@ rm -rf bin rendered tmp templates/grafonnet-lib
 ## Templates available
 
 The following templates are available:
-
-- API Performance Dashboard
-- Etcd Dashboard
-- OpenShift Performance Dashboard
-- OVN Dashboard
-- Pgbench Dashboard
-- UPerf Results Dashboard
-- Vegeta Dashboard
-- YCSB Dashboard
+ 
+- CPT
+    - [x] Ingress Perf Dashboard.
```

Comment:
can we just have checkboxes here? when we do a preview, a bullet point and then a checkbox appears which doesn't look that good.

Reply:
actually, this looks in this format:
![image](https://github.com/cloud-bulldozer/performance-dashboards/assets/88205562/6af327de-04db-455f-bbec-3945ff44d2e8)